compliance-report/generate.sh

```diff
@@ -18,6 +35,11 @@ See ${SARIF_NAME}
 Signature Information
 See ${AUTHORIZED_PUB_NAME}
 
+Security Report
+${SECURITY_REPORT}
+
 Known Vulnerabilities
 Any vulnerabilities that may be shown in the files referenced above have been reviewed and accepted by the appropriate approvers.
 EOF
+
+cat ${S3_ASSETS}/ssdlc_compliance_report.md
```

Comment:
Did you add this for debugging purposes or did you intend to print the generated report to STDOUT? Since this report is considered internal, I don't think we should leave this here.

Reply:
Yes this was for debugging, will remove